suite-native/transactions/src/components/TransactionDetail/TransactionDetailData.tsx

```diff
@@ -28,17 +37,24 @@ export const TransactionDetailData = ({
         selectTransactionBlockTimeById(state, transaction.txid, accountKey),
     );
 
+    const txn = useSelector((state: TransactionsRootState) =>
```

Comment:
nit:
```suggestion
    const transaction = useSelector((state: TransactionsRootState) =>
```

We are usually trying to keep the variable name aligned with the selector name.

Reply:
Btw is not this selected transaction exactly the same one, that is already stored in the `transaction` component prop?

Reply:
good catch. I needed different type, but the selector still needs to be recast.

Reply:
eaa9f838b